src/main/java/com/mixpanel/android/mpmetrics/MPDbAdapter.java

```diff
@@ -102,6 +102,13 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
             db.execSQL(PEOPLE_TIME_INDEX);
         }
 
+        public boolean belowMemThreshold() {
+            if (mDatabaseFile.exists()) {
+                return Math.min(mDatabaseFile.getUsableSpace(), 20 * 1024 * 1024) >= mDatabaseFile.length();
```

Comment:
It might be nice to put this constant (the minimum) in MPConfig - thats where configuration values can live, and it means you can give it a nice human readable name (and allow library users to adjust it)